apps/dashboard/app/javascript/xdmod/jobs.js

```diff
@@ -132,17 +163,48 @@ function tableRows(context, helpers) {
     td3.innerText = helpers.date(job);
 
     // <td>{{cpu_label cpu_user}}</td>
-    const td4 = document.createElement('td');
-    td4.innerHTML = helpers.cpu_label(job.cpu_user);
+    // Not used with new analytics data
+    // const td4 = document.createElement('td');
+    // td4.innerHTML = helpers.efficiency_label(job.cpu_user);
 
-    tr.append(td1, td2, td3, td4);
+    tr.append(td0, td1, td2, td3);
 
     rows.push(tr);
+
+    // Add job analytics placeholder
+    const analyticsRow = document.createElement('tr');
+    const analyticsData = analyticsTable('<td colspan="3">LOADING...</td>')
+    analyticsRow.innerHTML = `
+      <td colspan="4" class="hiddenRow">
+        <div class="collapse" id="details_${job.jobid}">
+          ${analyticsData}
+        </div>
+      </td>`;
+    rows.push(analyticsRow);
   });
 
   return rows;
 }
 
+function analyticsTable(analyticsContent) {
```

Comment:
I think nested tables like this are an accessibility issue. I.e., they're confusing to navigate through.

I'd guess we need an additional column before `id` column and within it just text.